packages/flutter/lib/src/rendering/table.dart

```diff
@@ -1034,6 +1053,8 @@ class RenderTable extends RenderBox {
     // then, lay out each row
     double rowTop = 0.0;
     for (int y = 0; y < rows; y += 1) {
+      if (y != 0)
```

Comment:
Presumably, this also needs changes to the intrinsics methods.

Reply:
If adding rowSpacing isn't a good idea, it might be better to close this PR and the related issue. 😂